examples/py/proxies-for-synchronous-python.py

```diff
@@ -0,0 +1,37 @@
+# The python version of the library uses the python-requests (https://python-requests.org) package for underlying HTTP and supports all means of customization available in the `requests` package, including proxies.
+
+import ccxt
+
+my_proxies = {
+    'http': 'http://1.2.3.4:5678',
+    'https': 'http://1.2.3.4:5678',
+}
+
+# You can pass them to the constructor
+exchange = ccxt.poloniex({
+    'proxies': my_proxies,
```

Comment:
is this format still available? according to the user's report it does not seem so

Reply:
I wanted to maintain this format (even though new format is preferred, where we also support callbacks).
so, if this format doesn't work, then I'll fix the bug